src/controllers/users.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const { UsersModel } = require('../models/users.js');
```

Comment:
```suggestion
const { usersModel } = require('../models/users.js');
```

It should be named with a lowercase first letter since it is an object, not class. And maybe would be more descriptive to rename `usersModel` to `userService`